### ISSUE TITLE
Add an "installation" to choco

### ISSUE THREAD

**Issue by arieljannai** [PFCKrutonium/Windows-10-Login-Background-Changer#79]
Hi!

**A feature request:**
It could be cool if there will be an installation of the program available in [chocolatey](https://chocolatey.org/)!
Of course it's not a real installation, it should only copy files and make a shortcut (run/desktop/both).

What do you think about it? :sunglasses: 

Ariel.

**Comment by arieljannai** [comment]
Self contained package, if needed:
https://github.com/chocolatey/choco/wiki/CreatePackages#self-contained

**Comment by emmatyping** [comment]
@PFCKrutonium @arieljannai I can start on this. I am already acquainted with chocolatey.

EDIT: While I am at it, I think I will compile an msi, because why not.....

**Comment by arieljannai** [comment]
> I can start on this. I am already acquainted with chocolatey.

Yay!

> While I am at it, I think I will compile an msi, because why not.....

Personally I prefer to drop the files and not install an application that I'm not going to use on a regular basis. But that just me :stuck_out_tongue: 
Anyhow, it's your and the developer choice :smile:

**Comment by Toyz** [comment]
Possible but what's the use of installing it either way? Just download the zip and make a shortcut to the extracted exe?

**Comment by arieljannai** [comment]
> Possible but what's the use of installing it either way? Just download the zip and make a shortcut to the extracted exe?

The installation itself via choco will give the benefits of updates through choco. So it will be updated when necessary when I'm looking for updates for all of my applications. I won't need to check here if there's an update.

We can use choco to create a package installation, which just drops the files in a folder and create a shortcut, or with an msi installer as @IronManMark20 suggested to make, so it will be installed as any other program. 

Because it's a standalone application, I personally think the package option is better.

**Comment by Toyz** [comment]
Or I could simplest write a internal updated that listens for a new release on git? Thus preventing third party software?

Thus removing unnecessary bloat

**Comment by Toyz** [comment]
Standalone/Portalable > Package

Throw it on a thumbdrive and walk away which is way better then installing it registering the files to windows which is what MSI does and other things

**Comment by Toyz** [comment]
Or we can be lazy and use https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/getting-started.md which is by github directly and intergrates into nuget also which creates a package but also does udpating and everything?

Simpler then using choco which requires choco to be installed and simpler then using a MSI because MSIs just suck

**Comment by emmatyping** [comment]
I agree that MSIs suck, but it seems they are the prefered method of installation for chocolatey. Portable apps "litter" the path just like MSIs with Chocolatey `The *.portable packages are the packages that will usually result in an executable on your path somewhere but do not get installed onto the system (Add/Remove Programs).` ([source](https://github.com/chocolatey/choco/wiki/ChocolateyFAQs#what-is-the-difference-between-packages-named-install-ie-autohotkeyinstall-portable-ie-autohotkeyportable-and--ie-autohotkey)), so yeah....

But I think I will just bundle the portable version. Makes it less intrusive. While Squirrel seems interesting, chocolatey has the advantage that it has a wide user base, and can be integrated into [OneGet](https://github.com/OneGet/oneget), the official apt-get like tool for Windows. 

Once I get the initial package, it really should be as simple as updating the version number.

**Comment by Toyz** [comment]
Squarral uses the .application format which is used by everything that runs .net cause ts a .net installer package even github for windows uses it in-case you didn't know that

**Comment by emmatyping** [comment]
Oh, thanks. Tbh, I think self updating would make the most sense. However, checking at the user's request (I'm thinking a button to update), makes more sense. Personally, I don't like things popping at me to ask for an update.

**Comment by Toyz** [comment]
Squarll can do that also

**Comment by emmatyping** [comment]
Even better, you seem more knowledgeable about this than I. I guess I will work on a french translation instead...

**Comment by Toyz** [comment]
We will give both out and see which is better, also use the editor i released to make the lang files and please use the language code as the file name and you can post your french translation on gist so we can get it

**Comment by arieljannai** [comment]
If we talk about translation - I can do a Hebrew one.
@Toyz Where can I find the your editor you mentioned?

**Comment by Toyz** [comment]
@arieljannai right there: https://github.com/Toyz/Language-XML-Editor/releases/tag/1.1
